benchmarks/dynamo/pr_time_benchmarks/benchmarks/aotdispatcher.py

```diff
@@ -0,0 +1,65 @@
+import sys
+
+from benchmark_base import BenchmarkBase
+
+import torch
+from torch.testing._internal.two_tensor import TwoTensor
+
+class Benchmark(BenchmarkBase):
+    def __init__(self, *, training, subclass):
+        self._training = training
+        self._subclass = subclass
+        self._device = "cpu"
+
+    def name(self):
+        prefix = f"aotdispatcher"
+        if self._training:
+            prefix += "_training"
+        else:
+            prefix += "_inference"
+        if self._subclass:
+            prefix += "_subclass"
+        else:
+            prefix += "_nosubclass"
+        if self._device == "cpu":
+            prefix += "_cpu"
+        return prefix
+
+    def description(self):
+        return "100 inputs, 100 outputs, each input is added once"
+
+    def _prepare_once(self):
+        args = [torch.ones(100, requires_grad=self._training, device=self._device) for _ in range(100)]
+        if self._subclass:
+            args = [TwoTensor(x, x.clone().detach().requires_grad_(self._training)) for x in args]
+        self.args = args
+
+    def _prepare(self):
+        torch._dynamo.reset()
+
+    def _work(self):
+        @torch.compile(backend="aot_eager_decomp_partition", fullgraph=True)
```

Comment:
do we reach any  inductor caches? or do we stop way beyond?
do we need      with fresh_inductor_cache()

Reply:
nope, the `aot_eager_decomp_partition` backend doesn't ever touch inductor so I removed the inductor cache code (although when the AOTDispatcher cache is eventually on by default we'll need to clear that cache here)